### PR TITLE
adjusting styling for small size buttons

### PR DESCRIPTION
**Pull request by faustoonna**
addressing new Design's requirement for small size buttons:
- Removing min-width;
- ensure that the button is 32px height;
- using small size icon for small size buttons

### CODE REVIEW

File: projects/pastanaga-angular/src/lib/styles/_buttons.scss [hunk 1]
```diff
@@ -39,7 +39,9 @@
         font-size: font-size(s);
         line-height: rhythm(3);
         padding: $padding-button-small;
-
+        &:not(.pa-button-icon):not(.pa-button-icon-text) {
+            min-width: auto;
+        }
```
Comment (mpellerin42) [inline projects/pastanaga-angular/src/lib/styles/_buttons.scss:44]: would be better to add a condition `:not(.pa-small)` where we set the min-width instead (line 34).

**Review by mpellerin42 — CHANGES_REQUESTED** [review]
whenever you do a PR on pastanaga, you need to update the version number in package.json and to add an entry corresponding to your change in CHANGELOG.md

File: projects/pastanaga-angular/src/lib/button/button.component.spec.ts [hunk 2]
```diff
@@ -24,9 +24,7 @@ describe('ButtonComponent', () => {
         expect(component._iconSize).toEqual(Size.large);
         component.size = Size.medium;
         expect(component._iconSize).toEqual(Size.medium);
-    });
-    it('should set the icon size to medium for small buttons', () => {
         component.size = Size.small;
-        expect(component._iconSize).toEqual(Size.medium);
+        expect(component._iconSize).toEqual(Size.small);
```
Comment (faustoonna) [inline projects/pastanaga-angular/src/lib/button/button.component.spec.ts:28]: @mpellerin42 do you know if we had any specific reason why we assigned the medium size icon to the small size button?

File: projects/pastanaga-angular/src/lib/styles/_buttons.scss (hunk 1 above)
Comment (faustoonna) [reply projects/pastanaga-angular/src/lib/styles/_buttons.scss]: done

File: projects/pastanaga-angular/src/lib/button/button.component.spec.ts (hunk 2 above)
Comment (mpellerin42) [reply projects/pastanaga-angular/src/lib/button/button.component.spec.ts:28]: it was designed that way on the first place by Glow.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# 2.36.18 (2021-08-31)
+### Improvements
+- **Button**: updating small size button styling
```
Comment (mpellerin42) [inline CHANGELOG.md:3]: you should add your github name into brackets next to your change. Pastanaga is an open source repo and it's a common practice in open source projects ;)

**Review by mpellerin42 — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (faustoonna) [reply CHANGELOG.md]: done

**Pull request merged**